src/docs/community/contributors/README.md

```diff
@@ -12,6 +12,8 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 !!! tip
     You can add new contributors by [using the All Contributors bot](https://allcontributors.org/docs/en/bot/usage).
 
+<!-- This is for the auto-generated All Contributors badge to point somewhere. -->
```

Comment:
maybe adding it at some point later then instead of adding it here?

Reply:
We need this outside the auto-generated `ALL-CONTRIBUTORS-BADGE` and `ALL-CONTRIBUTORS-LIST` sections. I figured that the badge linking to the top of the contributors list is the most sensible (we can't remove the badge pointing to the anchor, it'll always have this strange `#contributors-`).